lib/generators/rspec/integration/integration_generator.rb

```diff
@@ -10,12 +10,58 @@ class IntegrationGenerator < Base
                    :type => :boolean,
                    :default => true,
                    :desc => "Generate request specs"
+      class_option :api, :type => :boolean, :desc => "Creates request_spec for APIs, skip specs unnecessary for API-only apps"
+      class_option :fabrication, :type => :boolean, :desc => "Fill params with Fabricator model attributes"
+      class_option :factorybot, :type => :boolean, :desc => "Fill params with Factory Bot model attributes"
```

Comment:
These are best left to outside gems, we don't have any other integration with extensions like this, so we can't include this here.